storage/tokudb/ha_tokudb_alter.cc

```diff
@@ -23,25 +23,713 @@ Copyright (c) 2006, 2015, Percona and/or its affiliates. All rights reserved.
 
 #ident "Copyright (c) 2006, 2015, Percona and/or its affiliates. All rights reserved."
 
-#if TOKU_INCLUDE_ALTER_56
-
-#if 100000 <= MYSQL_VERSION_ID && MYSQL_VERSION_ID <= 100099
-#define TOKU_ALTER_RENAME ALTER_RENAME
-#elif (50600 <= MYSQL_VERSION_ID && MYSQL_VERSION_ID <= 50699) || \
-      (50700 <= MYSQL_VERSION_ID && MYSQL_VERSION_ID <= 50799)
 #define TOKU_ALTER_RENAME ALTER_RENAME
-#elif 50500 <= MYSQL_VERSION_ID && MYSQL_VERSION_ID <= 50599
-#define TOKU_ALTER_RENAME ALTER_RENAME_56
-#else
-#error
-#endif
 
-#include "ha_tokudb_alter_common.cc"
 #include <sql_array.h>
 #include <sql_base.h>
 
 #include <vector>
 
+TOKUDB_UNUSED(static bool tables_have_same_keys(TABLE* table,
```

Comment:
Remove TOKUDB_UNUSED, it's actually used?

Reply:
Ahh, maybe. I never considered these. Will have to validate each one.